gcp-data-drive/go.mod

```diff
@@ -11,8 +11,12 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-module gcpdatadrive
+module github.com/GoogleCloudPlatform/DIY-Tools/gcp-data-drive/gcpdatadrive
```

Comment:
Delete `gcpdatadrive` at the end so the module path matches the git path. The Go package will still be `gcpdatadrive`, it will just be imported with the dashes. For example, see https://github.com/google/go-cmp/blob/master/go.mod.

Reply:
I tried this suggestion but it seems to break appenging deployments so I rolled it back for now. I think I need to use the cmd path point feature in the appengine deploy to resolve but I will do it in another pull request. PHAL